packages/configraph/src/resolvers/pick.ts

```diff
@@ -15,8 +15,9 @@ export function createdPickedValueResolver(
       // since we handle resolution of services in the right order
       // we can assume the picked value will be resolved already (if it was possible at all)
       if (!sourceNode.isResolved) {
-        return new Error('picked value has not been resolved yet');
```

Comment:
it was returning and error as the result, which was getting serialized as `[Object object]`. I think it went unnoticed because before the parallelization work, the source items were almost always resolved by the time they got picked.